charts/nri-bundle/Chart.yaml

```diff
@@ -45,13 +45,21 @@ dependencies:
     version: 1.0.0
 
   - name: kube-state-metrics
-    version: 4.23.0
+    version: 4.25.0
```

Comment:
If we are taking into consideration upgrading a major would not make sense as well to drop the support for HPA metrics on old clusters <1.23?
I fear that we are making our life and configuration more complex for the HPA sample that most of the customers do not use.
In case they really need that they can always downgrade KSM and we can document that option on docs or even changing the image with the guided install. 

I am not sure we want to split the config to support EOLed version of K8s pinning a specific KSM version (with growing vulnerabilities over time).

Reply:
Of course I am fine with having this additional complexity, but I fear it is going to be a bad precedent. We cannot assume we are always supporting old clusters if the tools we use do not.
```
1.22
Latest Release:1.22.12 (released: 2022-07-13)
End of Life: 2022-10-28
```

If they run an old cluster they can set an old KSM image. We could provide a warning suggesting that.

https://github.com/kubernetes/kube-state-metrics/issues/1946

Reply:
Hi @paologallinaharbur, thanks for raising a really good point. No matter how much I want simplicity, we still have to continue to support the HPA metrics for our existing customers after discussing about this with the team.